tests/test_generics.py

```diff
@@ -530,8 +530,8 @@ def test_dynamic_serializer_form_in_browsable_api(self):
         view = DynamicSerializerView.as_view()
         request = factory.get('/')
         response = view(request).render()
-        self.assertContains(response, 'field_b')
-        self.assertNotContains(response, 'field_a')
+        assert response is 'field_b'
```

Comment:
Note that `assertContains` doesn't actually have a Python equivalent. It's a custom Django assertion that works based on the response content.